messages/en/vtex.admin-orders@0.x.json

```diff
@@ -0,0 +1,145 @@
+{
+  "admin.orders-menu-button": "All orders v2",
```

Comment:
these messages should be scoped by "admin/" so they don't end up in the store

Reply:
Updated.